Observer/FormConfig/Frontend/AddCheckoutBillingAddress.php

```diff
@@ -136,6 +136,8 @@ private function getActivePaymentMethodCodes(): array
             }
         }
 
+        $codes = "shared";
```

Comment:
This will overwrite whatever was stored in the array `$codes` with a string. Isn't appending to the array what we want to do instead?
```suggestion
        $codes[] = 'shared';
```